tests/system/test_system.py

```diff
@@ -2439,6 +2439,43 @@ def test_ubla_set_unset_preserves_acls(self):
         self.assertEqual(bucket_acl_before, bucket_acl_after)
         self.assertEqual(blob_acl_before, blob_acl_after)
 
+    def test_new_bucket_w_pap(self):
+        new_bucket_name = "new-w-pap" + unique_resource_id("-")
+        self.assertRaises(
+            exceptions.NotFound, Config.CLIENT.get_bucket, new_bucket_name
+        )
+        bucket = Config.CLIENT.bucket(new_bucket_name)
+        bucket.iam_configuration.uniform_bucket_level_access_enabled = True
+        bucket.create()
+        self.case_buckets_to_delete.append(new_bucket_name)
+
+        self.assertEqual(
+            bucket.iam_configuration.public_access_prevention, "unspecified"
+        )
+
+        bucket.iam_configuration.public_access_prevention = "enforced"
```

Comment:
Does there need to be a `bucket.patch` after this line?

Reply:
Yes, `bucket.patch` is what updates the bucket properties in the backend.